x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/discover/search_source_alert.ts

```diff
@@ -214,19 +214,19 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     await PageObjects.discover.selectIndexPattern(OUTPUT_DATA_VIEW);
 
-    let alertId: string;
+    let ruleId: string;
     if (type === 'name') {
       const [{ id }] = await getAlertsByName(value);
-      alertId = id;
+      ruleId = id;
```

Comment:
Do `alert_id` and `rule_id` hold the same values?

Reply:
No they aren't the same, the `rule_id` is the id of the rule and the `alert_id` is the id of the individual alert.